Replace player numbers with tree species and use player board images as background.

fonglh opened the pull request:
See https://boardgamegeek.com/thread/1840832/okay-so-what-are-trees-represented

sohymg commented:
i am not sure this is a good idea. makes it harder to associate the colour with the player?

fonglh commented:
I think the primary way people associate players in this game is by colour anyway. Using actual names gives a bit more visual differentiation in the modal vs `Player x`.

![image](https://user-images.githubusercontent.com/1902527/43354173-c7718a64-9279-11e8-8f44-13dc6a44bac4.png)

sohymg commented:
i mean in the sense when programming, perhaps add player constants like `const pine_player = 1`

fonglh committed:
Use player board images for background.

Add player board images.
Assign background image to each player.
Adjust background-position depending on the cell type.

fonglh commented:
<img width="706" alt="screen shot 2018-07-30 at 1 37 32 pm" src="https://user-images.githubusercontent.com/1902527/43379180-bdc199f2-93fd-11e8-85d4-400b2a9ecf10.png">

Backgrounds for modal selection, light point table cells and board cells.

fonglh commented on index.html:
```diff
@@ -132,13 +139,14 @@
       .select-player-modal .select-player-btn {
-        width: 150px;
-        height: 100px;
+        width: 146px;
```
Images are 1460px by 1050px.
